presto-cli/src/test/java/io/prestosql/cli/TestClientOptions.java

```diff
@@ -134,6 +136,15 @@ public void testSessionProperties()
         assertEquals(new ClientSessionProperty("foo="), new ClientSessionProperty(Optional.empty(), "foo", ""));
     }
 
+    @Test
+    public void testTimeZone()
+    {
+        ClientOptions options = new ClientOptions();
+        options.timeZone = "UTC";
```

Comment:
use a timezone that is never going to be tests' JVM zone
eg `Europe/Vilnius`

Reply:
Changed to Europe/Vilnius